rhaptos/cnxmlutils/xsl/cnxml-to-html5.xsl

```diff
@@ -42,6 +42,11 @@
         <xsl:value-of select="@class"/>
       </xsl:attribute>
     </xsl:if>
+    <xsl:if test="$version">
+        <xsl:attribute name="data-cnxml-html-ver">
+            <xsl:value-of select="$version"/>
+        </xsl:attribute>
+    </xsl:if>
```

Comment:
Comments on choice of attribute/position/name? Since I want to have access to this for the data-migration, I chose not to go the HTML comment route. (I don't believe in parsing comments, they are for humans)

Reply:
Including this version in the content is going to mean the content will change even if the actual transform did nothing. Maybe that's ok?

Reply:
Hmm, it does make diffing awkward (like including the PDF-gen timestamp). I guess the only other option for recording this would be to include the version in the DB, right?

I think that, by having this be an attribute, all content would have to be rebaked every time CNXML->HTML changed even if the particular content did not change. Is that correct?

Reply:
It's probably fine for now. If we have a little churn in the HTML that's no biggie (and it'll allow us to verify that the migration worked 😄 )

Also, I'd suggest `data-cnxml-to-html-ver` as the attribute to clarify that it's the conversion version